diff --git a/node_modules/pbasic-tokenizer/browser.js b/node_modules/pbasic-tokenizer/browser.js
index d5085c2..643524e 100644
--- a/node_modules/pbasic-tokenizer/browser.js
+++ b/node_modules/pbasic-tokenizer/browser.js
@@ -2572,7 +2572,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	      }};
 	  
 	  var NODEFS={isWindows:false,staticInit:function () {
-	        NODEFS.isWindows = !!process.platform.match(/^win/);
+	        NODEFS.isWindows = false;
 	      },mount:function (mount) {
 	        assert(ENVIRONMENT_IS_NODE);
 	        return NODEFS.createNode(null, '/', NODEFS.getMode(mount.opts.root), 0);
